lightkurve/lightcurve.py

```diff
@@ -949,7 +949,7 @@ def to_periodogram(self, nterms=1, nyquist_factor=1, oversample_factor=1,
                                            freq_unit=freq_unit,
                                            **kwargs)
 
-    def to_fits(self, path=None, overwrite=False, **extra_data):
+    def to_fits(self, path=None, overwrite=False, aperture_mask=None, **extra_data):
         """Writes the LightCurve to a FITS file.
 
         Parameters
```

Comment:
Please add an explanation of `aperture_mask` to the docstring as part of the `Parameters` section, clarifying that we accept both a boolean mask or an integer-mask, that an integer mask must always follow the Kepler/TESS convention, and that a boolean mask will be converted to an integer-mask when being stored in the FITS file.

Reply:
OK, I added the docstring info and shifted aperture mask in preparation for custom Tess/Kepler behavior in commit 2bf8e91.  Still TODO: add the function that parses a mask into the different TESS and Kepler conventions.

Reply:
OK, we can now accept a boolean mask, and convert it to a bitmask.